clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -304,6 +307,11 @@ func (p *pingChecker) run() {
 			}
 			// TODO(#685): Make ping checks less communication heavy.
 			pod, err := execservice.GetPod()
+			if err != nil {
+				success = 0
```

Comment:
I'm not very familiar with this code, but as a general rule, if you're swallowing the error it would be good to at least log it. Otherwise it would be really hard to debug if it stops working.

Could you add a klog.Warning somewhere here?

Reply:
Yeah that was a miss. Added it.